generators/server/templates/src/test/java/package/web/rest/UserJWTControllerIT.java.ejs

```diff
@@ -72,7 +72,8 @@ import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.header;
 import static org.hamcrest.Matchers.nullValue;
-import static org.hamcrest.Matchers.isEmptyString;
+import static org.hamcrest.Matchers.emptyString;
```

Comment:
Does isEmptyString is deprecated ? Why should we use is(emptyString()) instead ?

Reply:
yes it is deprecated http://hamcrest.org/JavaHamcrest/javadoc/2.0.0.0/org/hamcrest/text/IsEmptyString.html

Reply:
oh great ! My bad, I had read the wrong javadoc version ;)